src/routes/settings-usage-consolidation-route.js

```diff
@@ -56,12 +56,19 @@ const SettingsUsageConsolidationRoute = ({
       ...updatedData
     } = params;
 
+    const attributes = {
+      currency: 'USD',
+      platformType: updatedData.platformType,
+      startMonth: updatedData.startMonth,
+    };
```

Comment:
You missed `customerKey` value

Reply:
It's added later. This is because back-end sends this field as '*' characters. And if we include this field always even if it's not changed then the requests would fail because it's not a valid value.

Reply:
Sorry, I forgot about it, you're right